### ISSUE TITLE
[a11y] matchesSemantics is missing fields on SemanticsNode, such as transform

### ISSUE THREAD

**Issue by clocksmith** [flutter/flutter#49082]
Ideally, matchesSemantics has every field on SemanticsNode

**Comment by goderbauer** [comment]
cc @pdblasi-google
